src/SelectedStateParks/SelectedStateParks.js

```diff
@@ -19,18 +19,17 @@ function SelectedStateParks({ parks, selectedState, fetchParkDetails }) {
   return (
     <div className="card-grid">
       {filteredState.map((park) => {
-        const randomImage =
-          park.images[Math.floor(Math.random())];
+        const randomImage = park.attributes.media[Math.floor(Math.random())];
         return (
           <Card
             className="card"
-            id={park.id}
-            key={park.id}
-            states={park.states}
-            fullName={park.fullName}
+            id={park.attributes.park_code}
+            key={park.attributes.park_code}
+            states={park.attributes.states}
```

Comment:
again, its crazy that the refactor was this easy once we added the attribute tag